src/server/utils/onModuleLoad.js

```diff
@@ -64,6 +64,18 @@ function validateConfig(configValidators, config) {
 
 export const CONFIGURATION_KEY = 'appConfig';
 
+export function cspCheck(csp) {
+  if (!csp && !process.env.ONE_DANGER_DISABLE_CSP) {
+    throw new Error('Root module must provide a valid content security policy. If you would like to bypass this, set the ONE_DANGER_DISABLE_CSP enviornment variable to *, and NODE_ENV to development.');
```

Comment:
Should we only display this message in development ? Maybe add some logic to only add the second sentence "If you would like to bypass this, set the ONE_DANGER_DISABLE_CSP environment variable to *, and NODE_ENV to development." if NODE_ENV='development'.

```suggestion
    throw new Error('Root module must provide a valid content security policy. If you would like to bypass this, set the ONE_DANGER_DISABLE_CSP environment variable to *, and NODE_ENV to development.');
```

Reply:
I think this makes sense, makes the console less cluttered